add grainy bg image

steven4714 opened the pull request:
Added Grainy BG Image #5 
![Bildschirmfoto 2022-10-16 um 00 16 09](https://user-images.githubusercontent.com/46499853/196009591-628f3c80-5d9a-41b1-b75a-6a14a328829a.png)

steven4714 committed:
add grainy bg image

Waishnav commented on src/index.css:
```diff
@@ -5,6 +5,7 @@ body {
   font-family: 'Poppins', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url('../src/assets/grainy-effect.png');
```
```background: rgb(0, 0, 0) url("../src/assets/grainy-effect.png");```

Waishnav replied:
and delete background-color property

steven4714 committed:
change bg color and use shortcode

steven4714 commented:
Changed the color to 0,0,0 and deleted background-color property

![Bildschirmfoto 2022-10-17 um 08 58 02](https://user-images.githubusercontent.com/46499853/196109554-8ad9cd60-08a2-44ca-8d03-132eb1568351.png)
